lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -124,16 +129,224 @@ export class BrowserCacheManager extends CacheManager {
     }
 
     async initialize(correlationId: string): Promise<void> {
+        this.performanceClient.addFields(
+            {
+                cacheLocation: this.cacheConfig.cacheLocation,
```

Comment:
Please add these attributes to `PerformanceEvent` type.

Reply:
Done